src/Goerli-DssSpell.sol

```diff
@@ -50,79 +52,119 @@ contract DssSpellAction is DssAction {
+        {
+            bytes32 _ilk  = bytes32("GUSD-A");
+            address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_GUSD_A");
+            //
+            // Enable liquidations for GUSD-A
+            // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
+            //       the rely for consistency with other collaterals and in case the PIP
+            //       changes to an OSM.
+            DssExecLib.authorize(_clip, DssExecLib.clipperMom());
+            DssExecLib.setValue(_clip, "stopped", 0);
+            // Use Abacus/LinearDecrease
+            DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_GUSD_A);
+            // Set Liquidation Penalty to 0
+            DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
+            // Set Liquidation Ratio to 150%
+            DssExecLib.setIlkLiquidationRatio(_ilk, 15000);
+            // Set Auction Price Multiplier (buf) to 1
+            DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 10000);
+            // Set Local Liquidation Limit (ilk.hole) to 300k DAI
+            DssExecLib.setIlkMaxLiquidationAmount(_ilk, 300_000);
+            // Set tau for Abacus/LinearDecrease to 4,320,000 second
+            DssExecLib.setLinearDecrease(MCD_CLIP_CALC_GUSD_A, 4_320_000);
+            // Set Max Auction Duration (tail) to 43,200 seconds
+            DssExecLib.setAuctionTimeBeforeReset(_ilk, 43_200);
+            DssExecLib.setAuctionPermittedDrop(_ilk, 9900);
+            // Set Proportional Kick Incentive (chip) to 0
+            DssExecLib.setKeeperIncentivePercent(_ilk, 0);
+            // Set Flat Kick Incentive (tip) to 500
+            DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);
+            // Update spotter price
+            DssExecLib.updateCollateralPrice(_ilk);
+        }
+        {
+            bytes32 _ilk  = bytes32("USDC-A");
+            address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_USDC_A");
+            //
+            // Enable liquidations for USDC-A
+            // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
+            //       the rely for consistency with other collaterals and in case the PIP
+            //       changes to an OSM.
+            DssExecLib.authorize(_clip, DssExecLib.clipperMom());
+            DssExecLib.setValue(_clip, "stopped", 0);
+            // Use Abacus/LinearDecrease
+            DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_USDC_A);
+            // Set Liquidation Penalty to 0
+            DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
+            // Set Liquidation Ratio to 150%
+            DssExecLib.setIlkLiquidationRatio(_ilk, 15000);
+            // Set Auction Price Multiplier (buf) to 1
+            DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 10000);
+            // Set Local Liquidation Limit (ilk.hole) to 20m DAI
+            DssExecLib.setIlkMaxLiquidationAmount(_ilk, 20_000_000);
```

Comment:
Not saying we should use MILLION here but maybe next time we do smth like this? Looks fine either way